### DISCUSSION TITLE
Remove pagelabel only from showPagingButtons

### DISCUSSION THREAD

**Discussion by raimanshu** [stephanrauh/ngx-extended-pdf-viewer#1185] [Q&A]
Hii stephanrauh,

I was using a pdf in which first five page labels were 1 to 5

. Then when I was continue scrolling page label again starts from 1.

This was creating confusion for users of my project.

Is there any way to : **Remove page label input box from paging buttons and only show pages starts from 1 till end of pdf ** with total number of pages like 24 of 46 only

It will be a big help.

**Comment by stephanrauh** [comment]
Try this:
```typescript
  constructor() {
    pdfDefaultOptions.disablePageLabels = true;
  }
```
I haven't used this option myself yet, so I'd appreciate your feedback. Does this work as well as the documentation promises?

**Reply by raimanshu** [reply]
Thanks stephanrauh for your quick reply. I really appreciate your feedback.

**Reply by stephanrauh** [reply]
Did it solve your issue?

**Reply by raimanshu** [reply]
Yes. The problem is solved.. 👍
